SpriterDotNet/SpriterParser.cs

```diff
@@ -21,8 +22,8 @@ static SpriterParser()
 
         public static Spriter Parse(string data)
         {
+            if (data != null) data = data.Trim();
```

Comment:
Maybe better to replace the next line with IsNullOrWhiteSpace instead of allocating a new string here